website/common/locales/en/tasks.json

```diff
@@ -122,8 +122,8 @@
   "fortifyText": "Fortify will return all your tasks, except challenge tasks, to a neutral (yellow) state, as if you'd just added them, and top your Health off to full. This is great if all your red tasks are making the game too hard, or all your blue tasks are making the game too easy. If starting fresh sounds much more motivating, spend the Gems and catch a reprieve!",
   "confirmFortify": "Are you sure?",
   "fortifyComplete": "Fortify complete!",
-  "deleteTask": "Delete this Task",
-  "sureDelete": "Are you sure you want to delete this task?",
+  "deleteTask": "Delete this <%= type %>",
+  "sureDelete": "Are you sure you want to delete this <%= type %>?",
```

Comment:
@Shuyinsama This isn't a full review (Habitica's staff will do that later), but this is one thing I've noticed that I think will need to be changed. BTW, there's no way you could have known about this, so it's definitely not a criticism. :)

Can you please choose new names for the two new strings? E.g., `deleteTaskType` and `sureDeleteType` 
(It doesn't have to be those exact names, they're just examples.)

I.e., your pull request should still delete the old `deleteTask` and `sureDelete` strings but it should add two new strings with new names.

The rest of this comment is just an explanation for why this may be necessary. Feel free to ask for clarification if I don't explain it well!

The reason is related to our current translation process. When a string is changed in the English locales files, the English change is copied to Weblate (where translations are done) but the old strings in other languages are **not** automatically changed. The other languages have to be manually updated by volunteers, and it can be months before all languages are updated.

For simple changes in wording, that's not usually a problem, but when a variable is added to a string, it's important that all languages use the new string immediately. For any language that still has the old string, players using that language will see error messages in Habitica because the string doesn't have the variable.

So, the best way to ensure that all languages use the new string is to give the string a new name. The new string still may not be translated into all languages for several months, but any language that doesn't have the new string will use the English version. A new name ensures that players don't see errors from missing variables.

Reply:
👍